src/app/add-modules/add-modules.component.html

```diff
@@ -1,21 +1,28 @@
-<div fxLayout="column" fxLayoutGap="24px" class="add-module-container">
-  <div fxLayout="row" fxLayoutAlign="space-between center" class="crud-on-module">
-    <div class="module-text-container">
-      <p class="module-text">Modules</p>
-      <p class="module-hint">Select a module to edit, view, duplicate, change category or delete.</p>
+<div fxLayout="column" fxLayoutGap="18px" class="add-module pt-20">
+
+  <div fxLayout="row" fxLayoutAlign="space-between center" fxLayout.xs="column"
+       class="add-module__crud bg-white-color p-14">
+    <div class="add-module__text-container">
+      <p class="add-module__text font-size-20 text-light-black">Modules</p>
+      <p class="add-module__hint font-size-12 m-0 text-hint-color">Select a module to edit, view, duplicate, change
```

Comment:
for `p` tag there will be only margin top and bottom in default, u can use `my-0` only